pkg/reconciler/taskrun/taskrun_test.go

```diff
@@ -650,6 +701,17 @@ func createDynamicPoolHostConfig() []runtimeclient.Object {
 	return []runtimeclient.Object{&cm, &sec}
 }
 
+func createLocalHostConfig() []runtimeclient.Object {
+	cm := v1.ConfigMap{}
+	cm.Name = HostConfig
+	cm.Namespace = systemNamespace
+	cm.Labels = map[string]string{ConfigMapLabel: "hosts"}
+	cm.Data = map[string]string{
+		"local-platforms": "linux/arm64",
```

Comment:
let's use constant here if possible

Reply:
I tent to avoid that in tests to make sure I test the actual values and not constants that might be set to the wrong values.

Reply:
ok